.circleci/config.yml

```diff
@@ -215,6 +215,9 @@ jobs:
             #!/bin/bash
             VALE_STR_CLI_VERSION=3.3.0
 
+            # set smart sudo
+            if [[ $EUID -eq 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
```

Comment:
L25@https://circleci.com/developer/orbs/orb/circleci/vale#orb-source

Reply:
I'm wondering, any idea why we don't use the orb directly?